### ISSUE TITLE
Warrior Battleshout

### ISSUE THREAD

**Issue by DisdainThor** [Daeymien/SmartBuff_Unofficial#4]
The warrior shouts are group buffs again in BFA, I tried to change it, but now I just get the buff reminder and nothing happens when i use the mouse wheel. Any idea?

**Comment by Daeymien** [comment]
Notes for self after recent digging around:

Battle Shout is technically a self-buff as it's not targeted.

**Comment by jt198d** [comment]
Is it possible to add demon armor and amplify curse for warlocks?

**Comment by Daeymien** [comment]
@jt198d 

Yes, it is possible. However, I fail to see what that has to do with Warrior's Battle Shout.

Please do not hijack other issues.
